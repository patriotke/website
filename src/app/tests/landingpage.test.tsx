/* eslint-disable import/no-extraneous-dependencies */
import '@testing-library/jest-dom'
// eslint-disable-next-line @typescript-eslint/no-unused-vars
import { render, screen } from '@testing-library/react'
import LandingPage from '../page';

describe('LandingPage', () => {
    it('renders LandingPage', () => {
        render(<LandingPage />)

        const { container } = render(<LandingPage />)
        expect(container).toMatchSnapshot()
    })
})