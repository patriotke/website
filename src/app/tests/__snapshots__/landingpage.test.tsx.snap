// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage renders LandingPage 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-24"
  >
    Â 
  </main>
</div>
`;
